### DISCUSSION TITLE
2η Συμπληρωματική ΟΣΣ 5 + Ημερομηνία παράδοσης 5ης εργασίας

### DISCUSSION THREAD

**Discussion by Maurphal** [sdy60/syllabus#140] [Q&A] [answered]
Πότε θα γίνει η επόμενη ΟΣΣ? 
και πότε να θα παρουσιάσουμε αρθρο αν η παράδοση γίνει 8 Μαΐου? 
ευχαριστώ
Α.Δ.

**Comment by epidrome** [comment, marked as answer]
Η (προαιρετική) παρουσίαση του άρθρου μπορεί να γίνει την ημέρα της ΟΣΣ5, δλδ Κυριακή 12 Μαΐου ή σε συμπληρωματικές ΟΣΣ5 μετά την κανονική.

Η παράδοση της ΓΕ5 πρέπει να γίνει το αργότερο μέχρι την Τετάρτη 15 Μαϊου (έχει δωθεί παράταση σε όλους) ώστε να υπάρχει χρόνος για την βαθμολόγηση και προετοιμασία για την τελική γραπτή εξέταση.
